tests/foreman/sys/test_hooks.py

```diff
@@ -0,0 +1,159 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Foreman Hooks
+
+:Requirement: Foreman Hooks
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: SYS
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+
+"""
+from fauxfactory import gen_ipaddr
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo import ssh
+from robottelo.decorators import run_in_one_thread, tier3
+from robottelo.test import TestCase
+from robottelo.datafactory import valid_hosts_list, valid_hostgroups_list
+
+HOOKS_DIR = '/usr/share/foreman/config/hooks'
+LOGS_DIR = '/usr/share/foreman/tmp/hooks.log'
```

Comment:
This can be placed in constant.py ?

Reply:
It could but I'd still prefer to have it in a single file for convenience, as I don't believe these could be useful in any other test than this one